README.adoc

```diff
@@ -87,8 +87,8 @@ settings, other beans, and various property settings.
 such as setting up a `DispatcherServlet`. Spring Boot adds it  automatically when it sees
 `spring-webmvc` on the classpath.
 - `@ComponentScan`: Tells Spring to look for other components, configurations, and
-services in the the `com.example.testingweb` package, letting it find the
-`HelloController` class.
+services in the package where your annotated `TestingWebApplication` class resides
+(`com.example.testingweb`, letting it find the `com.example.testingweb.HelloController`).
```

Comment:
Change to 

(`com.example.testingweb`), letting it find the `com.example.testingweb.HelloController`.

(Note the position of the closing parenthesis.)

Reply:
resolved 96f0e94df495e081f1f763ea645a9bb7bb4e6de7